serenata_toolbox/federal_senate/dataset.py

```diff
@@ -25,9 +26,16 @@ def fetch(self):
             file_path = os.path.join(self.path, 'federal-senate-{}.csv'.format(year))
             try:
                 urlretrieve(url, file_path)
-            except Exception as exception:
-                print('While fetching Seranata Toolbox not found file: {} \n{}'.format(file_path, exception))
-                not_found_files.append(file_path)
+            except HTTPError as http_error_exception:
+                print('We failed to reach the server')
+                print('Error code  ', http_error_exception.reason)
+                print('While fetching Seranata Toolbox not found file: {} \n{}'.format(file_path, http_error_exception))
+                raise http_error_exception
+            except URLError as url_error_exception:
+                print('The server couldn\'t fulfill the request.')
```

Comment:
Following [PEP8 on string quotes](https://www.python.org/dev/peps/pep-0008/#string-quotes):
>When a string contains single or double quote characters, however, use the other one to avoid backslashes in the string

So let's remove this nasty backslash 😬